portal/src/components/Collections/Collection/Collection.component.js

```diff
@@ -29,17 +28,14 @@ export default {
     }
   },
   components: {
-    EditableContent,
     ShareCollection,
     DeleteCollection,
-    SwitchInput
+    SwitchInput,
+    InputWithCounter
   },
   mounted() {
-    const { collection } = this
-    if (!_.isEmpty(collection)) {
-      const title =
-        this.$i18n.locale === 'nl' ? collection.title_nl : collection.title_en
-      this.setCollectionTitle(title)
+    if (this.collection) {
```

Comment:
Ik ben hier wel voor een iets slimmere check. Als collection een empty object is dan val je hier doorheen. Ik overzie te weinig welke gevolgen dat kan hebben en welke vreemde bugs daaruit voort komen.

Reply:
Ik heb het aangepast. We gebruiken het component op 1 plek en hadden invloed op wat er in ging. Ik heb het aangepast, maar zorg er nu voor dat er geen leeg object het component in kan gaan. Ik ben ook voor defensief programmeren als het nodig is, maar zelfs in JS hebben we er soms nog wel wat invloed op 😉 .

Verder zie ik heel veel in codebase `import _ from 'lodash'`. Geen idee of je het weet, maar dat zorgt ervoor dat webpack heel lodash in de js bundle stopt. Het is beter om `import { isEmpty } from 'lodash'` te doen. Dan komt alleen de code die nodig is voor `isEmpty` in de JS bundle. Ik ben het namelijk al een paar keer tegengekomen. Het moet natuurlijk wel eerst overal weg voordat het effect heeft 😄 .

Reply:
Ja, ik weet het ondertussen, maar nog niet toen ik al die imports schreef. Ben bij andere projecten ook met omschrijven begonnen. Goed dat je het aanpakt.